### PR TITLE
[wip] GLPI 9.5 compatibility

### PR DESCRIPTION
**Pull request by cedric-anne**
TODO:

- [ ] handle migration from datetime to timestamp fields
- [ ] update tests to be compatible with usage of MYSQLI_OPT_INT_AND_FLOAT_NATIVE
- [ ] read domains informations of `Computer,` `NetworkEquipment` and `Printer` from `glpi_domains_items` table (instead of dropped  `domains_id` field)

### CODE REVIEW

**Comment by ddurieux** [comment]
Pleae, add in title the prefix `[wip]` if the PR is not ready to be reviewed ;)

File: inc/inventoryruleimport.class.php [hunk 1]
```diff
@@ -882,7 +882,8 @@ function findWithGlobalCriteria($input) {
                               && !is_null($value)
                               && is_numeric($value)
                               && $value > 0) {
-                           $this->criterias_results['found_port'] = $value;
+                              $this->criterias_results['found_port'] = $value;
```
Comment (ddurieux) [inline inc/inventoryruleimport.class.php]: The indentation was good, 3 spaces from the `if (` ;)

Comment (cedric-anne) [reply inc/inventoryruleimport.class.php]: indeed, but next line was missing (9.5 branch is currently broken)

**Comment by trasher** [comment]
See #2976

**Pull request closed without merging**